template/packages/frontend/.storybook/main.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  typescript: {
+    check: false,
+    checkOptions: {},
+    reactDocgen: "react-docgen-typescript",
+    reactDocgenTypescriptOptions: {
+      shouldExtractLiteralValuesFromEnum: true,
+      propFilter: prop =>
+        prop.parent ? !/node_modules/.test(prop.parent.fileName) : true,
+    },
+  },
+  addons: ["storybook-css-modules-preset"],
+  stories: ["../stories/*.stories.@(ts|tsx)"],
```

Comment:
If you use the recursive glob (`**`) stories don't have to be limited to the stories directory (but that can still be the convention), and we can put stories in subdirectories.

```suggestion
  stories: ["../**/*.stories.@(ts|tsx)"],
```